projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -129,6 +129,23 @@ export class IgxForOfDirective<T> implements OnInit, OnChanges, DoCheck, OnDestr
      */
     public totalItemCount: number = null;
 
+    /**
+     * A function that returns whether the element is the first or not
+     */
+    public first = (index) => { return index === 0};
```

Comment:
if these are in the context of forOf they should be affixed so that it's clear they return something regarding a forOf item, e.g.:

```typescript
private isFirstItem(index: number): boolean {
     return index === 0;
}
```

Notice that the function is also private, the parameters and return type is set. There is no reason for it to be public

Reply:
Done